### ISSUE TITLE
[FEAT] Color Picker tool

### ISSUE THREAD

**Issue by FuzzySadist** [Kruptein/PlanarAlly#1020]
**Is your feature request related to a problem? Please describe.**
I noticed when placing a sailing ship map, that I couldnt match the blue water that was on the map.  It would have been great to select "Simple" for the background, then color pick on the map water so that the map wasn't just sitting in the middle of a huge white area (or a mismatched blue area)

**Comment by develroo** [comment]
@Kruptein 

FWIW this should be an easy lift as all modern browsers have an `EyeDropper()` function, and it is up to the page to present it to the browser 


Web docs are here

https://developer.mozilla.org/en-US/docs/Web/API/EyeDropper

and here

https://developer.chrome.com/docs/capabilities/web-apis/eyedropper

**Comment by Kruptein** [comment]
You're absolutely right, it's just been on the backburner and one of those tickets I forgot about. Damn it's already been more than a year.

I should go through the issue log one of these days 😅

**Comment by develroo** [comment]
Actually, just found out it is not supported on Firefox. *bleah* But, no matter Chrome supports it. I'll have to file another issue against Firefox *sighs*

![Screenshot from 2024-01-29 10-19-05](https://github.com/Kruptein/PlanarAlly/assets/7777993/90d81450-f26b-4418-b62e-df3c7385b684)
